src/app/exhibitsPage/page.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link';
+import NavBar from '../../components/userComponents/navBar/navBar';
+import { CategoryRow } from '../../types/types';
+import { fetchAllCategories } from '../../supabase/category/queries';
+import Exhibit from '../../components/userComponents/Exhibit/Exhibit';
+import BackButton from '../../components/userComponents/BackButton/page';
+
+/**
+ * @returns exhibit page
+ */
+function App() {
+  const [exhibits, setExhibits] = useState<CategoryRow[]>([]);
+  useEffect(() => {
+    // Get exhibits
+    const getExhibits = async () => {
+      const fetchedExhibits: CategoryRow[] = await fetchAllCategories();
+      setExhibits(fetchedExhibits);
+    };
+    getExhibits();
+    // Detect the hash in URL and scroll to the element with the corresponding ID
+  }, [exhibits]);
+
+  useEffect(() => {
+    const { hash } = window.location;
+    if (hash) {
+      setTimeout(() => {
+        const element = document.querySelector(hash);
+        const yOffset = -50;
+        if (element) {
+          const y =
+            element.getBoundingClientRect().top + window.scrollY + yOffset;
+          // check on this offset later
+          window.scrollTo({ top: y, behavior: 'instant' });
+        }
+      }, 1000);
+    }
+  }, []);
+  return (
+    <div className="bg-ivory">
+      <NavBar />
+      <div className="p-4 m-auto">
+        <BackButton />
+        <div className="flex-col justify-start items-start mt-2">
+          <div className="text-night text-[32px] font-bold leading-9 font-['Lato'] mb-4">
+            Our Exhibits{' '}
+          </div>
```

Comment:
Use a text based tag (p, h1, h2, etc..) for abstraction purposes.  Use inspect on figma file to choose the right tag, review globals.css file for custom aliases.  Swithc text-[32px] to tailwind units  -> text-8 but there is probably a tag in globals.css that has the right styling